docs/integrations/apps/guide/catalyst-compatible-apps.mdx

```diff
@@ -1,28 +1,21 @@
 # Building Catalyst-Compatible Apps and Integrations
 
-If you have an existing app or integration in the BigCommerce App Marketplace that you offer for existing BigCommerce stores, it is possible that you will need to fork your app and make it Catalyst-compatible, or update your integration to support Catalyst. 
+If you have an existing app or integration in the BigCommerce App Marketplace that you offer for existing BigCommerce stores, it is possible that you will need to fork your app and make it Catalyst-compatible, or update your integration to support Catalyst. Prioritize integrations that enhance the shopper experience, as these have the greatest impact on site performance and usability.
```

Comment:
The wording of "fork" makes me uneasy, usually the work to be done is much less than a complete fork. I'm struggling figuring out the right verbiage though 😕 

```suggestion
If you have an existing app or integration in the BigCommerce App Marketplace that you offer for existing BigCommerce stores, it is possible that your app may not work out of the box with Catalyst. For example, if your app makes use of Script Manager, or Stencil Context Objects, your app will likely not function properly if loaded on a Catalyst storefront by default. Prioritize integrations that enhance the shopper experience, as these have the greatest impact on site performance and usability.
```

Reply:
Actually, I realize you elaborate on this further along in the document, maybe this comment is too nitpicky.